bootstrapping/misinformation.tarql

```diff
@@ -0,0 +1,20 @@
+base  <http://weverify.eu/resource/>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix puml:     <http://plantuml.com/ontology#>
+prefix schema: <http://schema.org/>
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix wev: <http://weverify.eu/ontology/>
+prefix geoname: <http://sws.geonames.org/>
+
+
+construct {
+    ?claimUri a schema:Claim ;
+              schema:text ?b ;
+} where {
+    bind (uri(concat("Claim/", "misinfomation/", ?a)) as ?claimUri)
```

Comment:
Same typo as in the other file- missing "r."

However it would be interesting to see whether we can't form the Claim URIs in the same way as in the convert.rq file. We expect most of these claims already appear in our database ingested from other sources and connecting them would be pretty neat.